package.json

```diff
@@ -39,7 +39,7 @@
     "rimraf": "^3.0.2",
     "ts-loader": "^6.2.2",
     "typescript": "^3.9.0-beta",
-    "vsce": "^1.74.0",
+    "vsce": "1.101.0",
```

Comment:
1. This package is deprecated https://www.npmjs.com/package/vsce
2. I don't see this dependency in VS Code - even back in 1.70. Can we just remove it completely? https://github.com/microsoft/vscode/blob/release/1.70/package.json

Reply:
The package is used in `gulpfile.js` to list files in the `gulp zip` task and to create the VSIX in the `gulp vsix` task.

Reply:
First off, I had a brain fart and didn't realize this was for the sqlite extension. So yeah ignore the second option 😆 

But yes, you should update to the vscode version of the package.